pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,10 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            data = base64.b64decode(api_repr.get('data', b''))
+        except (binascii.Error, TypeError):
+            data = api_repr.get('data', b'')
```

Comment:
OK, should I just make a util to decode_base64(), and then we can use it other places as well?